src/api/app/models/channel.rb

```diff
@@ -143,6 +143,8 @@ def branch_channel_package_into_project(project, comment = nil)
     tpkg.branch_from(cp.project.name, cp.name, nil, nil, comment)
     tpkg.sources_changed(wait_for_update: true)
 
```

Comment:
@adrianschroeter You need to disable the rubocop here to fix the test failure. Like this:

```
# rubocop:disable Rails/SkipsModelValidations
project.touch
# rubocop:enable Rails/SkipsModelValidations
```